JS/Lesson19/gulcin/recursion.js

```diff
@@ -108,6 +136,25 @@ const nestedObj = {
 		linkedIn: 'johndoe',
 	},
 };
+ function flattenObj(obj) {
+   const keys = Object.keys(obj);
+   const flattenObject = {};
+   if(keys.length === 0 || keys[0].length === 0) {
+	return '';
```

Comment:
Returning an empty object would be better in this case as we work with objects in this fucntion
```suggestion
	return {};
```